python/GangaLHCb/test/GPI/TestDataFiles.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from Ganga.testlib import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+
+class TestDataFiles(GangaUnitTest):
+
+    @external
+    def testDataFiles(self):
+
+        from Ganga.GPI import DiracFile, config
+
+        print config['LHCb']
+        # LFNs
+        name = 'test.txt'
+        lfn = DiracFile(lfn=name)
+        assert lfn.lfn == name
+        lfn = DiracFile('lfn:'+name)
+        assert lfn.lfn == name
+        lfn = DiracFile('LFN:'+name)
+        assert lfn.lfn == name
+        try:
+            lfn = DiracFile('pfn:'+name)
```

Comment:
My bad - I was thinking of `LHCbDataset`. So this works (or at least doesn't raise an exception) and gives:
```
[11:13:28]
Ganga In [3]: lfn = DiracFile('pfn:test.txt')

[11:13:30]
Ganga In [4]: lfn
Ganga Out [4]: 
 DiracFile (
   defaultSE = ,
   namePattern = pfn:test.txt,
   guid = ,
   remoteDir = ,
   compressed = False,
   localDir = ,
   lfn = ,
   failureReason = ,
   locations = []  ,
   subfiles =[]
 )
```

I don't know if this is right or not but that's where we are :smile: (at least as of 6.1.18)